packages/now-go/index.ts

```diff
@@ -53,12 +54,12 @@ export async function build({
   meta = {} as BuildParamsMeta,
 }: BuildParamsType) {
   if (process.env.GIT_CREDENTIALS && !meta.isDev) {
-    console.log('Initialize Git credentials...');
+    debug('Initialize Git credentials...');
     await initPrivateGit(process.env.GIT_CREDENTIALS);
   }
 
   if (process.env.GO111MODULE) {
-    console.log(`\nManually assigning 'GO111MODULE' is not recommended.
+    debug(`\nManually assigning 'GO111MODULE' is not recommended.
```

Comment:
```suggestion
    console.log(`\nManually assigning 'GO111MODULE' is not recommended.
```

I think this is worth leaving because it indicates a mistake